pkg/printers/internalversion/describe_test.go

```diff
@@ -2275,6 +2275,8 @@ Spec:
   Allowing ingress traffic:
     To Port: 80/TCP
     To Port: 82/TCP
+    From NamespaceSelector: id=app2,id2=app3
+         PodSelector: id=app2,id2=app3
```

Comment:
It would be better to use different labels here than in the other cases, just to guarantee it's using the right ones

Reply:
Fixed.